### PR TITLE
Create ray.json

### PR DESCRIPTION
**Pull request by rayhanadev**
My website!
Thank you for this awesome service!

### CODE REVIEW

**Comment by RayhanADev** [comment]
Ooh yay, I passed all checks!

File: domains/ray.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "RayhanADev's Website!",
+  "repo": "https://github.com/rayhanadev/site",
+  "owner": {
+    "username": "rayhanadev",
+    "email": "rayhan.arayilakath@wa-students.org"
+  },
+  "record": {
+    "CNAME": "its.rayhanadev.repl.co"
```
Comment (phenax) [inline domains/ray.json:9]: Can you confirm if `repl.co` allows CNAME records pointing to it? If not, I'd suggest either going with [github pages](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/creating-a-github-pages-site) or you can also use a URL redirection to your repl.

So either,

```suggestion
    "CNAME": "rayhanadev.github.io"
```

**OR**

```suggestion
    "URL": "https://its.rayhanadev.repl.co"
```

**Review by phenax — CHANGES_REQUESTED** [review]

**Comment by RayhanADev** [comment]
Oop, apparently it doesn’t :(
I think I’ll go with URL, thank’s for confirming!

File: domains/ray.json (hunk 1 above)
Comment (RayhanADev) [reply domains/ray.json]: Done!

**Review by phenax — APPROVED** [review]

**Pull request merged**